### PR TITLE
Further attempt to reduce flakiness

### PR DESCRIPTION
**Pull request by andybalaam**

### CODE REVIEW

File: platforms/web/cypress/e2e/clipboard/cut.spec.ts [hunk 1]
```diff
@@ -20,7 +20,8 @@ describe('Cut', () => {
     const editor = '.editor:not([disabled])[contenteditable="true"]';
 
     it('remove text that is cut to clipboard', { browser: 'electron' }, () => {
-        cy.visit('/').wait(300);
+        cy.visit('/');
+        cy.get(editor).wait(500);
```
Comment (florianduros) [inline platforms/web/cypress/e2e/clipboard/cut.spec.ts:24]: I wonder if we can wait until the `contentEditable` field is at true ?

```ts
cy.get('#editor',  { timeout: 1000 }).should('have.attr', 'contentEditable', true)
```

(I'm not familiar with the cypress syntax, so it should not work out of box ^^)

Comment (andybalaam) [reply platforms/web/cypress/e2e/clipboard/cut.spec.ts:24]: The `editor` variable already says that `[contenteditable="true"]` so it should not match the `get` until it is editable.

I don't know why some of the characters don't type :-(

Comment (andybalaam) [reply platforms/web/cypress/e2e/clipboard/cut.spec.ts:24]: Unless my CSS selector is wrong?

**Review by florianduros — APPROVED** [review]

File: platforms/web/cypress/e2e/clipboard/cut.spec.ts (hunk 1 above)
Comment (florianduros) [reply platforms/web/cypress/e2e/clipboard/cut.spec.ts:24]: The selectors seems to be good

Comment (andybalaam) [reply platforms/web/cypress/e2e/clipboard/cut.spec.ts:24]: Merging, in case it helps :-)

Comment (andybalaam) [reply platforms/web/cypress/e2e/clipboard/cut.spec.ts:24]: I did some local testing and it seems slightly more reliable so fingers crossed.

**Pull request merged**